### ISSUE TITLE
The problem of accuracy

### ISSUE THREAD

**Issue by duxiangcheng** [liuch37/sar-pytorch#5]
Thanks for code sharing.
Can the reproduced code achieve the results of the paper?

**Comment by liuch37** [comment]
I haven't tried the training set the authors used in the paper (by mixing Syn90k, SynthText and SynthAdd to a total of 6.4M patches).  I only tested the performance for individual dataset, with some small amount of sampling.
